languages.toml

```diff
@@ -489,6 +489,34 @@ args = { processId = "{0}" }
 name = "c-sharp"
 source = { git = "https://github.com/tree-sitter/tree-sitter-c-sharp", rev = "5b60f99545fea00a33bbfae5be956f684c4c69e2" }
 
+[[language]]
+name = "cel"
+scope = "source.cel"
+injection-regex = "cel"
+file-types = ["cel"]
+comment-token = "//"
+roots = []
```

Comment:
The empty `roots` keys can be removed since they're defaulted now (https://github.com/helix-editor/helix/pull/8803)